lib/modules/datasource/git-refs/readme.md

```diff
@@ -1,13 +1,15 @@
-This datasource can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
+You can use this datasource plus [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to update dependencies that are not supported by Renovate.
```

Comment:
The current sentence sounds like: "I can use `git-refs` to update _any_ dependency I want". `git-refs` probably only updates _one_ or a _few_ dependencies?

In any case we should explain _which_ dependencies you can update with `git-refs`. 😄